website/docs/docs/collaborate/column-level-lineage.md

```diff
@@ -20,6 +18,19 @@ dbt Cloud updates the lineage in Explorer after each run that's executed in the
 
 <LoomVideo id='3040bf2a2ade45eca7942a7aed6b730c' />
```

Comment:
Please remove, now outdated

Reply:
Done! it's included in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/5488/commits/6d409449d1b65a3b892200dc1c5a5753c0522eee)